src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
```

Comment:
test is prob failing because you use export default, you can use export here

Reply:
> test is prob failing because you use export default, you can use export here

Yes, the check failed and linter warns about it:
![image](https://github.com/mate-academy/react_tabs-js/assets/63243136/3822d19e-3188-41f5-adc2-6835fd059743)